api/can_api_v2_definition.py

```diff
@@ -7,7 +7,7 @@
 from libs.datasets import timeseries
 import pydantic
 import datetime
-from covidactnow.datapublic.common_fields import GetByValueMixin
+from datapublic.common_fields import GetByValueMixin
```

Comment:
As we evolve this codebase we should probably try to unify the code.  Having a "datapublic" package really doesn't make any sense anymore.  Not for this PR though!

Reply:
Yeah I agree it does feel a bit disjointed